assets/helpers/abTests/abtestDefinitions.js

```diff
@@ -18,4 +19,39 @@ export const tests: Tests = {
     independent: true,
     seed: 3,
   },
+  newPaymentFlow: {
+    variants: ['control'],
+    audiences: {
+      ALL: {
+        offset: 0,
+        size: 1,
+      },
+    },
+    // TODO: turn to false
```

Comment:
when?

Reply:
before I merge

Reply:
it was for my testing

Reply:
probably before merging, if we are to go ahead with this. Then when we launch the test we have a another PR switching it to true.